pkgs/top-level/python-packages.nix

```diff
@@ -16229,24 +16229,13 @@ in {
   };
 
   pygit2 = buildPythonPackage rec {
-    name = "pygit2-0.25.1";
+    name = "pygit2-${pkgs.libgit2.version}";
```

Comment:
This is clever: the version will be kept in sync. Since its a `fetchzip`, the `src.name` will contain the version and the store path will change even if somebody updates `libgit2` version but forgets to update `pygit2` hash.

Reply:
Well, it will break as the hash no longer matches so nothing stays silent.